Adding Color Space documentation

PatrickRyanMS opened the pull request:
Adding documentation to authoring assets section on how to use color space in Babylon.js. Includes text and images for doc.

closes #774

PatrickRyanMS committed:
Adding text and images for Color Space Docs

PatrickRyanMS committed:
Merge branch 'master' of https://github.com/PatrickRyanMS/Documentation

RaananW reviewed (approved):
LGTM! just two comments before merging

PatrickRyanMS committed:
Update h tags in text and h styles in scss

Updated headers to have the correct h tag level for ToC indention. Also updated scss for h tags to have more contrast  and visual balance between the levels. Increased the size of h5 and h6 to not be smaller than p tag text as well as giving some styling emphasis.

PatrickRyanMS committed:
Moved styles to markdown container

I moved the styles for h tags to the markdown container to prevent spoiling of the styling of other page elements outside the markdown. Also found a page (sceneOptimizer.md) on random testing that had hr tags before every h tag resulting in rules above and below the header. Removed the hr tags as they were no longer needed.
